tests/trainers/test_monte_carlo.py

```diff
@@ -0,0 +1,27 @@
+
+from abmarl.sim.corridor import MultiCorridor
+from abmarl.managers import AllStepManager
+from abmarl.sim.wrappers import RavelDiscreteWrapper
+from abmarl.policies.q_table_policy import EpsilonSoftPolicy
+from abmarl.trainers.monte_carlo import OnPolicyMonteCarloTrainer
+
+
+def test_epsilon_soft():
+    sim = AllStepManager(RavelDiscreteWrapper(MultiCorridor(num_agents=1)))
+    ref_agent = sim.agents['agent0']
+    policy = EpsilonSoftPolicy(
+        action_space=ref_agent.action_space,
+        observation_space=ref_agent.observation_space
+    )
+    trainer = OnPolicyMonteCarloTrainer(sim=sim, policy=policy)
+    trainer.train(iterations=10_000, horizon=20)
+
+    policy.epsilon = 0
+    obs = sim.reset()
+    for _ in range(20):
+        # TODO: Enable no-explore mode for testing performance of policy.
```

Comment:
This may be resolved as part of combining EpsilonSoft with Greedy Policy, in which case we can have an arg in the Trainer for explore = False, or we can do what we do here with policy.epsilon = 0

Reply:
The way it is now is okay. Make this into another ticket to prototype a no_explore arg to see if that helps the interface.

Reply:
#245